src/components/UserBlog.js

```diff
@@ -34,7 +35,7 @@ export default function UserBlog({
   updatedAt,
 }) {
   const [expanded, setExpanded] = React.useState(false);
-  const [responseSuccessData,setResponseSuccessData] = React.useState(false);
+  const [responseSuccessData, setResponseSuccessData] = React.useState(false);
```

Comment:
Not needed !!,  Just a suggestion before creating PR check for all the changes , you had made.